src/mono/browser/runtime/startup.ts

```diff
@@ -168,31 +169,43 @@ function preInit(userPreInit: (() => void)[]) {
 }
 
 async function preInitWorkerAsync() {
-    mono_log_debug("worker initializing essential C exports and APIs");
+    if (!MonoWasmThreads) return;
```

Comment:
How can this happen?

Reply:
this is just to trim, tree-shake the code. I don't think it would be called.